in_toto/in_toto_mock.py

```diff
@@ -31,8 +31,21 @@
   in-toto-mock --name write-code -- touch foo.py
   ```
 
-"""
+<Arguments>
+  name:
+          A unique name to relate mock link metadata with a step defined
+          in the layout.
+  command:
+          A list where the first element is a command and the remaining
+          elements are arguments passed to that command.
+
+
+<Return Codes>
+  2 if an exception occurred during argument parsing
```

Comment:
Where is `2` returned in the code?

Reply:
By `ArgumentParser.error`, which can called by `argparse` internally when parsing errors, and I call it explicitly too (see 4ae540139e0831350ea49782ecd956a2ad3480d8).